docs/start/myths/overlays.md

```diff
@@ -5,16 +5,52 @@ parent: Debunking myths
+
+### The Distinction Between Overlays and Toolbars
+
+While both accessibility overlays and user preference toolbars may offer similar features, their underlying functionality and purpose are fundamentally different. Understanding this distinction is crucial for effective web accessibility.
+
+* **Accessibility Overlays:** These are presented as a comprehensive solution to accessibility challenges. They are designed to act as a "fix" for a website's underlying code by attempting to automatically remediate accessibility issues on the fly. However, they frequently fail to correct fundamental problems with a site’s HTML, ARIA, or CSS. This approach can create new barriers for users of assistive technologies and often provides a false sense of compliance. An overlay is intended to fix the site for the user, but it cannot fix inaccessible source code.
+
+* **User Preference Toolbars:** In contrast, toolbars are client-side applications that do not alter the website's original code. Their purpose is not to "fix" a site but to empower the end-user. They offer personal customization features, allowing individuals to modify their own browsing experience to suit their needs, such as adjusting text size or contrast. These tools are most effective when used on a website that is already built to be accessible, as they complement, rather than substitute for, proper accessibility practices.
+
+### Additional Resources
+
+For a more in-depth understanding, we recommend consulting the following resources from prominent experts and organizations in the accessibility community.
+
+#### The Overlays Fact Sheet
+
+A comprehensive, multi-lingual report detailing the issues and risks associated with accessibility overlays:
+* [overlayfactsheet.com](https://overlayfactsheet.com/en/)
+
+#### Videos from Accessibility Experts
+
+* **"Overlays Underwhelm"** by Adrian Roselli: A detailed presentation on the shortcomings of overlays.
+    * [https://www.youtube.com/live/RDkqJaBwgKU](https://www.youtube.com/live/RDkqJaBwgKU)
+
+* **"There is no spoon"** by Léonie Watson: This well-known talk discusses the importance of designing for people, not for technology, and the limitations of quick-fix solutions.
+    * [https://www.youtube.com/watch?v=fyRxd072JrA](https://www.youtube.com/watch?v=fyRxd072JrA)
+    (Note: The video you originally linked is about AI and accessibility, but this is Léonie Watson's celebrated talk on this topic.)
+
+#### Resources from the WordPress Community
+
+* **Equalize Digital's Analysis:** The WordPress accessibility agency Equalize Digital, in collaboration with Karl Groves, provides a detailed analysis titled **"Accessibility Overlays - All Sizzle, No Steak."**
```

Comment:
For all next text:

URLs as link texts are not meaningful and a pain to listen to by screen reader users.
Link for example the title of the page you are linking to the "Accessibility Overlays – All Sizzle, No Steak: Karl Groves" and not the URL.

Remove all list items.